README.md

```diff
@@ -1,22 +1,181 @@
-## Steps to use this Template effectively
+# BookStore_React
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍